content/implementations.md

```diff
@@ -0,0 +1,81 @@
+---
+doc-note-type: draft
+layout: standalone_resource
+title: "Accessibility Test Tools & Methodologies"
+permalink: /standards-guidelines/act/implementations/
+ref: /standards-guidelines/act/implementations/
+lang: en
+type_of_guidance: false
+footer: ""
+github:
+  repository: w3c/wcag-act-rules
+  path: content/implementations.md
+---
+
+{%- assign siteData = site.data.wcag-act-rules %}
+{%- assign implementations = siteData.act-implementations | sort: "name" %}
+
+{::nomarkdown} {% include box.html type="start" title="Summary" %} {:/}
+ACT rules provide a standard way to compare the coverage of accessibility test tools and methodologies. This pages shows how many rules different tools and methodologies have accurately implemented. Scores are based on public test data.
+{::nomarkdown} {% include box.html type="end" %} {:/}
+
+## Test Methodologies
+
+Methodologies provide step by step instructions on how to test accessibility standards.
+See [understanding ACT consistency](#understandingactconsistency).
+
+{% assign methodologies = implementations | where: "type", "Test methodology" %}
+{% include_relative _implementations-table.html implementations=methodologies %}
+
+## Semi-automated Test Tools
+
+Tools that combine user input and automated testing to test accessibility standards.
+See [understanding ACT consistency](#understandingactconsistency).
+
+{% assign semiAutoTools = implementations | where: "type", "Semi-automated tool" %}
+{% include_relative _implementations-table.html implementations=semiAutoTools %}
+
+## Automatic Test Tools
+
+Tools that fully automatically test accessibility standards.
+See [understanding ACT consistency](#understandingactconsistency).
+
+{% assign automatedTools = implementations | where: "type", "Automated tool" %}
+{% include_relative _implementations-table.html implementations=automatedTools %}
```

Comment:
In the reports column there is an icon that identifies each of the reports. It has an alt text but that is not visible on screen. Should we at least have it visible on hover?